cypress/integration/features/login_managment/forgotPasswordDailog.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Visibility of dBildungscloud Password Reset pop-up
```

Comment:
The folder name is not correct:

IS: login_managment
SHOULD: login_management